view/frontend/web/js/view/payment/method-renderer/razorpay-method.js

```diff
@@ -60,7 +60,7 @@ define(
             },
 
             initObservable: function() {
-                var self = this;
+                this._super();              //Resolves UI Error on Checkout
```

Comment:
@seherkdc what exactly was the reason for the error and how does `this._super()` resolves it

Reply:
@ccrims0n Refer Dev Docs for [uiClass library](https://devdocs.magento.com/guides/v2.1/ui_comp_guide/concepts/ui_comp_uiclass_concept.html) page. 
The _super() method calls the parent UI component method with the same name as the _super() method’s caller; if that method does not exist in the parent UI component, then the method tries to find it higher in the inheritance chain.